zulip/integrations/bridge_with_irc/irc_mirror_backend.py

```diff
@@ -21,6 +20,8 @@ def __init__(self, zulip_client, stream, topic, channel,
         self.nickserv_password = nickserv_password
         # Make sure the bot is subscribed to the stream
         self.check_subscription_or_die()
+        # Connect to bot after proper connection to Zulip server has been confirmed.
+        irc.bot.SingleServerIRCBot.__init__(self, [(server, port)], nickname, nickname)
```

Comment:
This change should probably in its own commit.

Reply:
I have split the commit in two.